src/Microsoft.CSharp/tests/CSharpGetIndexBinderTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace Microsoft.CSharp.RuntimeBinder.Tests
+{
+    public class CSharpGetIndexBinderTests
+    {
+        [Fact]
+        public void ArrayIndexing()
+        {
+            dynamic d = new[] { 0, 1, 2, 3 };
+            for (int i = 0; i != 4; ++i)
+            {
+                Assert.Equal(i, d[i]);
+            }
+        }
```

Comment:
Well, they're only a tiny part of the expressions project test, and they'd only be a small part of the CSharp project once more specific tests were added to that, so neither project seems to be the sole place to put these.
If a separate test project was included in both projects (so someone working on either could see what was already being checked), would a full CI run run it once, or twice.